crnk-gen/crnk-gen-asciidoc/src/main/java/io/crnk/gen/asciidoc/capture/AsciidocCaptureModule.java

```diff
@@ -290,6 +291,58 @@ public void delete(I id) {
 		}
 	}
 
+	@Deprecated
```

Comment:
you can remove the deprecator, it is perfectly valid to implement both oneRelationship repository and manyRelationship repository. Looks good otherwise. Maybe it would make sense to have this combining one, many wrapper in crnk-core next to WrappedRelationshipRepository as the use case comes up in other places.

Reply:
Thanks. I made some changes and move combine wrapper to package with wrappers.